frontend/public/Bee.json

```diff
@@ -0,0 +1 @@
],[75.394,37.779],[-3.483,5.01],[-79.348,-38.694]],"v":[[-153.52,-81.288],[-49.681,-33.942],[153.557,81.037],[-50.481,-24.074]]},"ix":2}},{"ty":"fl","bm":0,"hd":false,"mn":"ADBE Vector Graphic - Fill","nm":"Fond 1","c":{"a":0,"k":[1,1,1],"ix":4},"r":1,"o":{"a":0,"k":100,"ix":5}},{"ty":"tr","a":{"a":0,"k":[0,0],"ix":1},"s":{"a":0,"k":[100,100],"ix":3},"sk":{"a":0,"k":0,"ix":4},"p":{"a":0,"k":[1288.78,1316.99],"ix":2},"r":{"a":0,"k":0,"ix":6},"sa":{"a":0,"k":0,"ix":5},"o":{"a":0,"k":100,"ix":7}}]},{"ty":"gr","bm":0,"hd":false,"mn":"ADBE Vector Group","nm":"Groupe 2","ix":2,"cix":2,"np":4,"it":[{"ty":"sh","bm":0,"hd":false,"mn":"ADBE Vector Shape - Group","nm":"TracÃ© 1","ix":1,"d":1,"ks":{"a":0,"k":{"c":true,"i":[[1.567,3.149],[-87.804,35.399],[-0.808,-3.122],[89.002,-36.988]],"o":[[0,0],[85.405,-34.433],[1.637,6.332],[-89.003,36.988]],"v":[[-169.803,68.454],[-0.877,-3.987],[169.779,-68.47],[6.701,3.584]]},"ix":2}},{"ty":"fl","bm":0,"hd":false,"mn":"ADBE Vector Graphic - Fill","nm":"Fond 1","c":{"a":0,"k":[1,1,1],"ix":4},"r":1,"o":{"a":0,"k":100,"ix":5}},{"ty":"tr","a":{"a":0,"k":[0,0],"ix":1},"s":{"a":0,"k":[100,100],"ix":3},"sk":{"a":0,"k":0,"ix":4},"p":{"a":0,"k":[1326.52,1132.98],"ix":2},"r":{"a":0,"k":0,"ix":6},"sa":{"a":0,"k":0,"ix":5},"o":{"a":0,"k":100,"ix":7}}]},{"ty":"gr","bm":0,"hd":false,"mn":"ADBE Vector Group","nm":"Groupe 3","ix":3,"cix":2,"np":4,"it":[{"ty":"sh","bm":0,"hd":false,"mn":"ADBE Vector Shape - Group","nm":"TracÃ© 1","ix":1,"d":1,"ks":{"a":0,"k":{"c":true,"i":[[-10.611,-51.11],[-117.128,39.343],[34.318,68.963],[106.605,-49.998]],"o":[[7.719,37.182],[111.619,-37.492],[-35.547,-71.434],[-79.751,37.404]],"v":[[-204.811,81.828],[48.755,131.837],[197.04,-73.98],[-52.286,-119.367]]},"ix":2}},{"ty":"fl","bm":0,"hd":false,"mn":"ADBE Vector Graphic - Fill","nm":"Fond 1","c":{"a":0,"k":[0.5725,0.7804,0.7608],"ix":4},"r":1,"o":{"a":0,"k":100,"ix":5}},{"ty":"tr","a":{"a":0,"k":[0,0],"ix":1},"s":{"a":0,"k":[100,100],"ix":3},"sk":{"a":0,"k":0,"ix":4},"p":{"a":0,"k":[1323.99,1128],"ix":2},"r":{"a":0,"k":0,"ix":6},"sa":{"a":0,"k":0,"ix":5},"o":{"a":0,"k":100,"ix":7}}]},{"ty":"gr","bm":0,"hd":false,"mn":"ADBE Vector Group","nm":"Groupe 4","ix":4,"cix":2,"np":4,"it":[{"ty":"sh","bm":0,"hd":false,"mn":"ADBE Vector Shape - Group","nm":"TracÃ© 1","ix":1,"d":1,"ks":{"a":0,"k":{"c":true,"i":[[26.998,-39.03],[-99.34,-52.449],[-47.745,63.005],[99.757,38.84]],"o":[[-19.641,28.394],[94.666,49.982],[43.813,-57.816],[-74.628,-29.056]],"v":[[-173.703,-101.841],[-57.88,99.185],[164.16,81.254],[49.409,-112.753]]},"ix":2}},{"ty":"fl","bm":0,"hd":false,"mn":"ADBE Vector Graphic - Fill","nm":"Fond 1","c":{"a":0,"k":[0.5725,0.7804,0.7608],"ix":4},"r":1,"o":{"a":0,"k":100,"ix":5}},{"ty":"tr","a":{"a":0,"k":[0,0],"ix":1},"s":{"a":0,"k":[100,100],"ix":3},"sk":{"a":0,"k":0,"ix":4},"p":{"a":0,"k":[1283.565,1321.472],"ix":2},"r":{"a":0,"k":0,"ix":6},"sa":{"a":0,"k":0,"ix":5},"o":{"a":0,"k":100,"ix":7}}]}],"ind":17}],"v":"5.9.0","fr":25,"op":50,"ip":0,"assets":[]}
```

Comment:
For a better project structure, I suggest you to create subfolders.
For this file I would put it in `public/animation/spelling-bee/bee.json`
And don't use capital letter in the file name, we should keep that for components or hooks